lib/dao.js

```diff
@@ -2824,6 +2824,14 @@ function(data, options, cb) {
 
   var allowExtendedOperators = connector.settings &&
     connector.settings.allowExtendedOperators;
+  if (options.hasOwnProperty('allowExtendedOperators')) {
+    allowExtendedOperators = options.allowExtendedOperators === true;
```

Comment:
I think it may be better to allow the connector to prevent `options.allowExtendedOperators` to enable this feature. 

```js
if (allowExtendedOperators !== false) {
  if (options.hasOwnProperty('allowExtendedOperators')) {
    allowExtendedOperators = options.allowExtendedOperators === true;
  } 
  // ...
}
```


@fabien thoughts?